prov/efa/src/rxr/rxr_pkt_entry.h

```diff
@@ -79,15 +79,15 @@ struct rxr_pkt_entry {
 #endif
 };
 
+#if defined(static_assert) && defined(__x86_64__)
```

Comment:
Why is this change in this patch? Doesn't seem to do anything other than move if def randomly

Reply:
Umm, must have been a rebase artifact. Will fix this up.